src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -26,7 +26,6 @@ import (
 	_metrics "github.com/pachyderm/pachyderm/src/server/pkg/metrics"
 
 	"github.com/spf13/cobra"
-	"go.pedge.io/pkg/cobra"
```

Comment:
Nice ... I think that's the last instance of this lib reference

Reply:
It may well be, if it is I'll remove the library from vendor as part of this PR.